app/bundles/DashboardBundle/Event/WidgetDetailEvent.php

```diff
@@ -254,14 +307,6 @@ public function getTranslator()
         return $this->translator;
     }
 
-    /**
-     * Set security object to check the perimissions.
-     */
-    public function setSecurity(CorePermissions $security): void
```

Comment:
Please, revert this change as some plugin can call this public method. Mark it as deprecated instead.